File: Aprendendo-C++/Exemplos/tipo.cpp
#include <iostream>
#include <iomanip>
#include <string>

using namespace std;

int main(){
    int idade;
    double salario, altura;
    char genero;
    string nome;

    idade = 20;
    salario = 5800.50;
    altura = 1.76;
    genero = 'f';
    nome = "Gabs";

    cout << fixed << setprecision(2);
    cout << "nome = " << nome << endl;
    cout << "idade = " << idade << endl;
    cout << "altura = " << altura << endl;
    cout << "salario = " << salario << endl;
    cout << "genero = " << genero << endl;

return 0;
}

File: AprendendoCShap/Exemplos/tipo.cs
using System;
using System.Globalization;

namespace tipo {
    class Program
    {
        static void Main(string[] args){

            CultureInfo Ci = CultureInfo.InvariantCulture;

            int idade;
            double salario, altura;
            char genero;
            string nome;

            idade = 22;
            salario = 9000.0;
            altura = 1.79;
            genero = 'm';
            nome = "Gabriel Martins";

            System.Console.WriteLine(idade);
            System.Console.WriteLine( salario);
            System.Console.WriteLine(altura);
            System.Console.WriteLine(genero);
            System.Console.WriteLine(nome);


        }
        
    }

}